templates/horizon/bin/kolla_theme_setup

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -x
+
+SRC_CUSTOM_THEME_DIR=${1:-/etc/openstack-dashboard/theme}
+TARGET_THEME_DIR=${2:-/usr/share/openstack-dashboard/openstack_dashboard/themes}
+TAR_OPTS="-xzvf"
+
+function unpack_theme {
+  local theme="$1"
+  if [[ "$theme" == *.tar.gz ]]; then
+      echo "Unpacking theme ..."
+      tar "${TAR_OPTS}" "${SRC_CUSTOM_THEME_DIR}/${theme}" -C "${TARGET_THEME_DIR}"
+  else
+      echo "$theme is not valid, ignoring ..."
```

Comment:
Maybe give a more specific error message to help the user if they didn't read the doc?
```
echo "$theme does not end in tar.gz, ignoring ..."
```